generic/tests/netperf.py

```diff
@@ -254,8 +286,13 @@ def _pin_vm_threads(vm, node):
         logfile.write(output)
         logfile.close()
 
+    mtu_default = 1500
+    error_context.context("Change back server, client and host's mtu to %s"
+                          % mtu_default)
+    mtu_set(mtu_default)
+
```

Comment:
Same as above, add if before mtu_set(mtu_default)

Reply:
I have updated it, please help review.